Jenkinsfile

```diff
@@ -0,0 +1,37 @@
+pipeline {
+    agent none
+    /*
+    environment {
+        BRANCH_LOWER=BRANCH_NAME.toLowerCase()
+        VANITY_URL="${BRANCH_LOWER}.pharmanetenrolment-dqszvc-dev.pathfinder.gov.bc.ca"
+        FRONTEND_ARGS="-p HTTP_PORT=8080 -p HTTP_SCHEMA=http TERMINATION_TYPE='Edge' -p REDIRECT_URL=http://${VANITY_URL} -p VANITY_URL=${VANITY_URL}"
+        API_ARGS="-p ASPNETCORE_ENVIRONMENT=Development -p HTTP_PORT=8080 -p HTTP_SCHEMA=http -p VANITY_URL=${VANITY_URL} -p REDIRECT_URL=http://${VANITY_URL}"
+    }*/
+    options {
+        disableResume()
+    }
+    stages {
+        stage('Build Branch') {
+            options {
+                timeout(time: 90, unit: 'MINUTES')   // timeout on this stage
```

Comment:
should we abstract this to a variable?

Reply:
Could do but different stages can have different timing.  Build is usually the slowest.